config/initializers/carrierwave.rb

```diff
@@ -17,3 +17,16 @@
     }
   end
 end
+
+# FIX for CVE-2023-49090
+module CarrierWave
+  module Uploader
+    module ContentTypeWhitelist
+      private
+
+      def whitelisted_content_type?(content_type)
+        Array(content_type_whitelist).any? { |item| content_type =~ /\A#{item}/ }
```

Comment:
Is there any possibility of content_type being nil? Maybe `to_s` if we're unsure?

When I monkey-patch, I sometimes raise an exception if the gem version isn't exactly the version that we're monkey-patching.

Reply:
@yakloinsteak does line 22 do what you were thinking?   It appears to error if the version doesn't match.

Reply:
Thanks!